src/browser/api/window.js

```diff
@@ -837,6 +838,40 @@ Window.create = function(id, opts) {
     const { plugins } = manifest || {};
     winObj.plugins = JSON.parse(JSON.stringify(plugins || []));
 
+    webContents.on('console-message', (thing, level, message, lineNo, sourceId) => {
+        const manifestUrl = coreState.getConfigUrlByUuid(identity.uuid);
+        if (!manifestUrl) {
+            electronApp.vlog(2, `Error: could not get manifest url for app with uuid: ${identity.uuid}`);
+        }
+
+        function checkPrependLeadingZero(num) {
+            let str = String(num);
+            if (str.length === 1) {
+                str = '0' + str;
+            }
+
+            return str;
+        }
+
+        const date = new Date();
+        const month = checkPrependLeadingZero(date.getMonth() + 1);
+        const day = checkPrependLeadingZero(date.getDate());
+        const year = String(date.getFullYear()).slice(2);
+        const hour = checkPrependLeadingZero(date.getHours());
+        const minute = checkPrependLeadingZero(date.getMinutes());
+        const second = checkPrependLeadingZero(date.getSeconds());
+
+        // Format timestamp to match debug.log
+        const timeStamp = '' + month + '/' + day + '/' + year + ' ' + hour + ':' + minute + ':' + second;
```

Comment:
Open to suggestions on this - couldn't find an easy way to format the time

Reply:
I'd suggest ```const timeStamp = `${month}/${day}/${year} ${hour}:${minute}:${second}`;```